get_request.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+
+@app.route('/get', methods=['GET'])
+def get_request():
+    name = request.args.get('name', 'Guest')
```

Comment:
What does this function do?

Reply:
The `if __name__ == '__main__':` line checks if the script is run directly (not imported as a module). If it is, it executes `app.run(debug=True)`, which starts the Flask application with debug mode enabled, allowing for live debugging and reloading during development.